lib/src/controls/better_player_controls_state.dart

```diff
@@ -363,7 +403,7 @@ abstract class BetterPlayerControlsState<T extends StatefulWidget>
   }
 
   Widget _buildAudioTrackRow(
-      BetterPlayerHlsAudioTrack audioTrack, bool isSelected) {
+      dynamic audioTrack, bool isSelected) {
```

Comment:
I do not prefer dynamic here. Maybe you can pass text here?

Reply:
Solved with the unification